ra/ra.go

```diff
@@ -357,6 +363,10 @@ func NewValidationProfiles(defaultName string, configs map[string]ValidationProf
 			return nil, fmt.Errorf("ValidAuthzLifetime value must be greater than 0 and less than 398d, but got %q", config.ValidAuthzLifetime.Duration)
 		}
 
+		if config.MaxNames <= 0 || config.MaxNames > 100 {
```

Comment:
Optional: The latter case can be removed, it's being checked by `max=100` on line `299`.


```suggestion
		if config.MaxNames <= 0 {
```

Reply:
Keeping this as-is for now, during the transition between the two config formats. Once the transition is complete and there's only one source-of-truth for this value, removing the belt-and-suspenders check feels safer.